runtime/ftplugin/help.lua

```diff
@@ -1,7 +1,13 @@
 -- use treesitter over syntax (for highlighted code blocks)
 vim.treesitter.start()
 
--- add custom highlights for list in `:h highlight-groups`
+-- Higlight "===" and "---" heading delimiters specially.
+vim.cmd[[
```

Comment:
Any reason not to use the Lua highlight API here?

Reply:
Or just define these highlight groups in the default scheme?

Reply:
> Or just define these highlight groups in the default scheme?

didn't know about `@markup.heading.1.delimiter.vimdoc` , thanks

> Any reason not to use the Lua highlight API here?

more verbose. someone else can do that if they want